src/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import FilterButton from './components/FilterButton';
+
+const DATA = (JSON.parse(localStorage.getItem('todos')) || [
+  { id: "todo-0", name: "Sugondese", completed: false },
+  { id: "todo-1", name: "Ligma", completed: false },
+  { id: "todo-2", name: "Joe", completed: false }
+]);
+
```

Comment:
usually if something is not just a constant, you should not do all caps on it.

All caps usually refers to it as static data.